### PR TITLE
build: dont release beta docs

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-XYZ |
| :--------------------: | :--------: |

## 🧰 Changes

Ope

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -47,17 +47,17 @@ jobs:
...
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch: next
-        continue-on-error: true
+      #- name: Sync to next
+      #if: "github.ref == 'refs/heads/main'"
+      #uses: ad-m/github-push-action@master
+      #with:
+      #github_token: ${{ secrets.GITHUB_TOKEN }}
+      #branch: ${{ github.ref }}
+      #continue-on-error: true
```
Comment (jennspencer) [inline .github/workflows/release.yml:56]: is losing the indenting an issue?

Comment (kellyjosephprice) [reply .github/workflows/release.yml:56]: Oh, I don't know why my editor does that.

Comment (jennspencer) [reply .github/workflows/release.yml]: i mean it probably doesn't matter since it's commented out, heh

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.47</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.47)
<br>

**Pull request merged**